### ISSUE TITLE
Learning rate in pretraining on CIFAR10

### ISSUE THREAD

**Issue by Yeon07** [AndrewAtanov/simclr-pytorch#9]
Hi, thanks for sharing your implementation. It's very helpful.
If I've understood the your implementation right, it seems like you're training with the learning rate initially set to 4.0 on CIFAR10 . 
May I ask why you set it to 4.0? 
It seems like the authors of simCLR use one out of {0.5, 1.0, 1.5} (not sure which one though), so I'm quite confused. 
I would be very thankful if you could explain the reasons behind choosing 4.0 as your learning rate. 

Thanks in advance :)

**Comment by Yeon07** [comment]
Ah, understood. Thank you :)
